README.rst

```diff
@@ -360,6 +360,27 @@ Methods:
     a celery's task for example or to handle errors.
 
 
+Settings
+=============
+
+You can configure Django-Templated-Email by setting the following settings
+
+.. code-block:: python
+
+    TEMPLATED_EMAIL_FROM_EMAIL      # String containing the email to send the email from - fallback to DEFAULT_FROM_EMAIL  
+    TEMPLATED_EMAIL_BACKEND         # The backend that will send the email  
```

Comment:
`# The backend that will send the email` -> `# The backend class that will send the email, as a string like 'foo.bar.TemplateBackend' or the class reference itself`

Reply:
Do we have a default? If so, please specify

Reply:
Updated and added default value